platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/custom_tasks.adoc

```diff
@@ -13,92 +13,115 @@
 // limitations under the License.
 
 [[custom_tasks]]
-= Developing Custom Gradle Task Types
+= Developing Custom Task Types
 
-Gradle supports two types of task. One such type is the simple task, where you define the task with an action closure. We have seen these in <<tutorial_using_tasks.adoc#tutorial_using_tasks,Build Script Basics>>. For this type of task, the action closure determines the behaviour of the task. This type of task is good for implementing one-off tasks in your build script.
+Gradle supports two types of tasks.
 
-The other type of task is the enhanced task, where the behaviour is built into the task, and the task provides some properties which you can use to configure the behaviour. We have seen these in <<more_about_tasks.adoc#more_about_tasks,Authoring Tasks>>. Most Gradle plugins use enhanced tasks. With enhanced tasks, you don't need to implement the task behaviour as you do with simple tasks. You simply declare the task and configure the task using its properties. In this way, enhanced tasks let you reuse a piece of behaviour in many different places, possibly across different builds.
+One such type is the *simple task*, where you define the task with an action closure.
+For this type of task, the action closure determines the behavior of the task.
+This type of task is good for implementing one-off tasks in your build script.
 
-The behaviour and properties of an enhanced task are defined by the task's class. When you declare an enhanced task, you specify the type, or class of the task.
+The other type of task is the *enhanced task*, where the behavior is built into the task, and the task provides some properties that you can use to configure the behavior.
```

Comment:
🤔 I never paid attention to that "enhanced task" naming, but with the new writing, I would advise to replace it with something that clarifies we expect tasks like that to have a dedicated _type_.

Proposal:
> The other type of task is based on a *task type*, where the behavior is built into the task type, and it provides some properties that you can use to configure the behavior.

Reply:
Although from looking at the rest of the text, this might be hard to do.

Reply:
I updated it to "custom type task".